src/main.rs

```diff
@@ -49,139 +49,198 @@ fn run() -> Result<(), failure::Error> {
                             .short("e")
                             .long("expiration")
                             .takes_value(true)
                             .value_name("SECONDS")
-                            .help("the time, measured in number of seconds since the UNIX epoch, at which the entries should expire"),
                         )
                         .arg(
                             Arg::with_name("expiration-ttl")
+                            .help("Number of seconds for which the entries should be visible before they expire. At least 60")
                             .short("t")
                             .long("ttl")
                             .value_name("SECONDS")
                             .takes_value(true)
-                            .help("the number of seconds for which the entries should be visible before they expire. At least 60"),
                         )
                         .arg(
                             Arg::with_name("file")
+                            .help("The value passed in is a filename; open and upload its contents")
                             .short("f")
                             .long("file")
                             .takes_value(false)
-                            .help("the value passed in is a filename; open and upload its contents"),
                         )
                 )
                 .subcommand(
-                    SubCommand::with_name("write-bulk")
-                        .about("upload multiple key-value pairs at once")
+                    SubCommand::with_name("get")
+                        .about("Get a key's value from a namespace")
                         .arg(
                             Arg::with_name("namespace-id")
-                                .help("the id of your Workers KV namespace")
-                                .required(true)
-                                .index(1),
+                            .help("The ID of the namespace this action applies to")
+                            .required(true)
+                            // .short("n")
+                            // .long("namespace-id")
+                            // .value_name("<ID>")
+                            // .takes_value(true)
                         )
                         .arg(
-                            Arg::with_name("path")
-                            .help("the json file of key-value pairs to upload, in form [{\"key\":..., \"value\":...}\"...].")
+                            Arg::with_name("key")
+                            .help("Key whose value to get")
                             .required(true)
-                            .index(2),
                         )
                 )
                 .subcommand(
-                    SubCommand::with_name("delete-key")
+                    SubCommand::with_name("delete")
+                        .about("Delete a key (and its value) from a namespace")
```

Comment:
i think the parens are unnecessary
```suggestion
                        .about("Delete a key and its value from a namespace")
```